_posts/2020-Q1-open-source-report.markdown

```diff
@@ -0,0 +1,153 @@
+---
+layout: post
+title: "2020 Q1 open-source-report"
```

Comment:
@bronzdoc This continues to look a little weird. Why not just use a title like this? 

```suggestion
title: "Open Source Report (2020 Q1)"
```